web/src/lib/components/admin-page/settings/storage-template/storage-template-settings.svelte

```diff
@@ -84,7 +84,28 @@
   };
 </script>
 
-<section class="dark:text-immich-dark-fg">
+<section class="dark:text-immich-dark-fg mt-2">
+  <form autocomplete="off" on:submit|preventDefault class="mx-4 flex flex-col gap-4 py-4">
+    <div in:fade={{ duration: 500 }}>
```

Comment:
This form isn't needed, the classes can be moved to the div

Reply:
The stutter is caused by loading the storage template options from the server, it will wait for a response and then render the settings.

https://github.com/immich-app/immich/blob/3cd3411c1f5f5d9fd14218f31a391b3ed9351bae/web/src/lib/components/admin-page/settings/storage-template/storage-template-settings.svelte#L88

Reply:
And for the original comment, you can just push a new commit no need to create a new PR

Reply:
Ah thanks makes sense. About the PR I meant toss it in with another issue that I'll pick up. Anyways, I'll just commit it here

Reply:
@michelheusschen committed, have a look